### ISSUE TITLE
[issue] golang/protobuf warning

### ISSUE THREAD

**Issue by bigangryrobot** [megaease/easegress#36]
Receiving this warning message when executing the server code locally
```
WARNING: Package "github.com/golang/protobuf/protoc-gen-go/generator" is deprecated.
	A future release of golang/protobuf will delete this package,
	which has long been excluded from the compatibility promise.
```

**Comment by benja-wu** [comment]
Thanks for reminding this. As running the `go mod why` check, seems that
```bash
√ easegress $ go mod why github.com/golang/protobuf/protoc-gen-go/generator
# github.com/golang/protobuf/protoc-gen-go/generator
github.com/megaease/easegress/pkg/cluster
go.etcd.io/etcd/embed
github.com/grpc-ecosystem/grpc-gateway/runtime
github.com/golang/protobuf/protoc-gen-go/generator
```
this warning is caused by `grpc-gateway` in `etcd`. In this [release-note](https://github.com/grpc-ecosystem/grpc-gateway/releases?after=v2.0.0-beta.5), `grpc-gatway` project had removed this dependence in 1.14.5. 
We will figure out which latest version could be stable and also been able to remove this warning.

**Comment by xxx7xxxx** [comment]
This is because etcd[1] used grpc-gateway[2] needs the package golang/protobuf. Fixed by https://github.com/grpc-ecosystem/grpc-gateway/pull/1260 

The upgrade[3] of etcd will fix it.

[1] https://github.com/etcd-io/etcd/issues/12124
[2] https://github.com/grpc-ecosystem/grpc-gateway
[3] https://github.com/etcd-io/etcd/blob/main/CHANGELOG-3.5.md#package-clientv3

**Comment by benja-wu** [comment]
Merged by #38
